courses/basic-algorithm-scripting/get-index-to-ins.js

```diff
@@ -0,0 +1,14 @@
+
```

Comment:
I think that each function needs JSDocs, for a better understanding of what this want to achieve 😕